### PR TITLE
updates to codeowners

### PR DESCRIPTION
**Pull request by turadg**
## Description

- Adds an owner for /vat-data I had added (per https://github.com/Agoric/agoric-sdk/pull/4963#issuecomment-1083296820)
- Adds an owner for /web-components (don't know why this didn't have one)
- Remove the redundant catch-all wildcard (all paths are under `/`)

### CODE REVIEW

File: .github/CODEOWNERS
Comment (michaelfig) [inline .github/CODEOWNERS:5]: My intent in proposing this line was to catch only the files in the root of the repository.  If it does more than that, I'm happy to remove it, otherwise, let's leave it in.

**Review by michaelfig — APPROVED** [review]
Approved modulo a check for the semantics of `/*`.

File: .github/CODEOWNERS
Comment (turadg) [reply .github/CODEOWNERS:5]: Ah. My IDE says that even a file like `/packages/who.txt` is owned by this list.

How about `/*.*`? (EDIT: done)

Comment (michaelfig) [reply .github/CODEOWNERS:5]: I think your IDE may be incorrect.  Both [Gitlab](https://docs.gitlab.com/ee/user/project/code_owners.html#example-codeowners-file) and [Github CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) say something to this effect:

```
# End a path in `/*` to specify Code Owners for every file in
# a directory, but not nested deeper. This code matches
# `docs/index.md` but not `docs/projects/index.md`:
/docs/* @root-docs
```

Comment (turadg) [reply .github/CODEOWNERS:5]: Aha, yes. Also corroborated by https://github.com/hmarr/codeowners  

https://github.com/Agoric/agoric-sdk/pull/4966 and uninstalled the buggy extension =)

**Pull request merged**
